.github/CONTRIBUTING.md

```diff
@@ -51,7 +51,6 @@ If generate has already run, then it does not need to run again.
 ```sh
 go build
 go fmt ./...
-golint ./...
```

Comment:
maybe `go vet` to use the builtin?

Reply:
From a few minutes of Googling around, I'm not sure if it is a 1:1 in what passes it makes.  I was reading more about it in this comment: https://github.com/golangci/golangci-lint/issues/1892#issuecomment-841546395.  `revive` advertises itself as a drop in replacement but looks like we need a config file of sorts.  `golangci-lint` has IDE support, CI support, and can use `revive` (although disabled by default) in its linters.  I can try `go vet`, `revive`, and `golangci-lint` out and see what happens.

Reply:
i don't know that we were necessarily relying on anything for lint though. so vet or one of the others is just fine.